mlrun/model_monitoring/stores/models/__init__.py

```diff
@@ -11,13 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
+
+from typing import Optional, Type
+
+from .mysql import ModelEndpointsTable as MySQLModelEndpointsTable
+from .sqlite import ModelEndpointsTable as SQLiteModelEndpointsTable
 
 
-def get_ModelEndpointsTable(connection_string: str = None):
+def get_model_endpoints_table(connection_string: Optional[str] = None) -> Type:
```

Comment:
I suggest to change/remove the `Type` typehint because it raises a lot of warning messages within `sql_model_endpoint_store.py` and doesn't provide any actual important information about the returned value.

Reply:
I agree. The situation in VSCode was better than in PyCharm.
I improved the type hint which solved the issue in PyCharm and added two waivers for VSCode.
3f9d7a7